_includes/block-by-numbers.html

```diff
@@ -0,0 +1,24 @@
+{% assign team_by_number = site.data.our_team.team_by_number %}
```

Comment:
It looks like you forgot to include the data file. Since this is just the theme repository, please put some test data here.